install_config/aggregate_logging.adoc

```diff
@@ -12,628 +12,482 @@ toc::[]
+|Description
+
+|*_kibana.crt_*
+|A browser-facing certificate for the Kibana server.
+
+|*_kibana.key_*
+|A key to be used with the Kibana certificate.
+
+|*_kibana-ops.crt_*
+|A browser-facing certificate for the Ops Kibana server.
+
+|*_kibana-ops.key_*
+|A key to be used with the Ops Kibana certificate.
+
+|*_server-tls.json_*
+|JSON TLS options to override the Kibana server defaults. Refer to
+https://nodejs.org/api/tls.html#tls_tls_connect_options_callback[Node.JS] docs
+for available options and the default options for an example.
+
+|*_ca.crt_*
+|A certificate for a CA that will be used to sign all certificates generated by
+the deployer.
+
+|*_ca.key_*
+|A matching CA key.
+|===
 +
-To allow *td-agent* access to the containers logs, create the
-*_/etc/sysconfig/td-agent_* file and ensure it contains the following:
+You can supply the files when creating a new secret:
 +
-====
 ----
-DAEMON_ARGS=
-TD_AGENT_ARGS="/usr/sbin/td-agent --log /var/log/td-agent/td-agent.log --use-v1-config"
+$ oc secrets new logging-deployer \
+   kibana.crt=/path/to/cert kibana.key=/path/to/key
 ----
-====
 
-. Add the following line to the *_/etc/td-agent/td-agent.conf_* file:
+. Create the deployer link:../admin_guide/service_accounts.html[service
+account]:
 +
 ====
 ----
-  @include config.d/*.conf
+$ oc create -f - <<API
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: logging-deployer
+secrets:
+- name: logging-deployer
+API
 ----
 ====
-
-. Create the *_/etc/td-agent/config.d/kubernetes.conf_* file with the following
-contents:
++
+Then, ensure the logging-deployer can edit the service. The following example
+uses the `logging` project. Be sure to use the project created above in the
+following example:
 +
 ====
-
-[source,xml]
 ----
-    <source>
-      type tail
-      path /var/lib/docker/containers/*/*-json.log
-      pos_file /var/log/td-agent/tmp/fluentd-docker.pos
-      time_format %Y-%m-%dT%H:%M:%S
-      tag docker.*
-      format json
-      read_from_head true
-    </source>
+$ oc policy add-role-to-user edit \
+          system:serviceaccount:logging:logging-deployer
+----
+====
 
-    <match docker.var.lib.docker.containers.*.*.log>
-      type kubernetes
-      container_id ${tag_parts[5]}
-      tag docker.${name}
-    </match>
+== Deploying the EFK Stack
 
-    <match kubernetes>
-      type copy
-      <store>
-        type forward
-        send_timeout 60s
-        recover_wait 10s
-        heartbeat_interval 1s
-        phi_threshold 16
-        hard_timeout 60s
-        log_level trace
-        require_ack_response true
-        heartbeat_type tcp
-        <server>
-          name logging_name <1>
-          host host_name <2>
-          port 24224
-          weight 60
-        </server>
+The EFK stack is deployed link:../dev_guide/templates.html[using a template].
+This ensures all future project configurations will feature the EFK stack. The
+default repository and branch of the template are set to origin/master. Change
```

Comment:
@adellape @sosiouxme I'm not too sure if I understand the question... I tried looking in the enterprise and origin template and couldn't find the answer.